resources/views/welcome-employee.blade.php

```diff
@@ -0,0 +1,39 @@
+@extends ('layouts.main_employee')
```

Comment:
This shouldn't be a specific page yet, we just need it to indicate that the login was successful.
1. Remove the timelog section
2. Remove the calendar scripts and CSS copied from the admin page